scalardl-test/src/main/java/scalardl/Common.java

```diff
@@ -81,4 +81,13 @@ public static Retry getRetryWithExponentialBackoff(String name) {
 
     return Retry.of(name, retryConfig);
   }
```

Comment:
To reduce the code redundancy, can you please also change the other `getRetryWithFixedWaitDuration()` to use the new one as follows:
```java
  public static Retry getRetryWithExponentialBackoff(String name) {
    return getRetryWithExponentialBackoff(name, MAX_RETRIES, SLEEP_BASE_MILLIS);
  }
```

Reply:
Thank you for the suggestion.
I will change the other getRetryWithFixedWaitDuration() to use the new one as mentioned in the comment.